3.6results

```diff
@@ -0,0 +1,860 @@
+builtins: 'function' is in stub but is not defined at runtime
+calendar: 'LocaleType' is in stub but is not defined at runtime
+calendar: 'calendar' is in __all__ but not in stub
+calendar: 'firstweekday' is in __all__ but not in stub
+calendar: 'month' is in __all__ but not in stub
+calendar: 'monthcalendar' is in __all__ but not in stub
+calendar: 'prcal' is in __all__ but not in stub
+calendar: 'prmonth' is in __all__ but not in stub
+calendar: 'weekheader' is in __all__ but not in stub
+configparser: 'ConverterMapping' is in __all__ but not in stub
+configparser: 'LegacyInterpolation' is in __all__ but not in stub
+configparser: 'MAX_INTERPOLATION_DEPTH' is in __all__ but not in stub
+configparser: 'SafeConfigParser' is in __all__ but not in stub
+configparser: 'SectionProxy' is in __all__ but not in stub
+csv: '__doc__' is in __all__ but not in stub
+csv: '__version__' is in __all__ but not in stub
+datetime: 'TimeTuple' is in stub but is not defined at runtime
+difflib: 'Match' is in __all__ but not in stub
+difflib: 'diff_bytes' is in __all__ but not in stub
+doctest: 'COMPARISON_FLAGS' is in __all__ but not in stub
+doctest: 'DONT_ACCEPT_BLANKLINE' is in __all__ but not in stub
+doctest: 'DONT_ACCEPT_TRUE_FOR_1' is in __all__ but not in stub
+doctest: 'DebugRunner' is in __all__ but not in stub
+doctest: 'DocFileSuite' is in __all__ but not in stub
+doctest: 'DocTest' is in __all__ but not in stub
+doctest: 'DocTestFailure' is in __all__ but not in stub
+doctest: 'DocTestFinder' is in __all__ but not in stub
+doctest: 'DocTestParser' is in __all__ but not in stub
+doctest: 'DocTestRunner' is in __all__ but not in stub
+doctest: 'DocTestSuite' is in __all__ but not in stub
+doctest: 'ELLIPSIS' is in __all__ but not in stub
+doctest: 'Example' is in __all__ but not in stub
+doctest: 'FAIL_FAST' is in __all__ but not in stub
+doctest: 'IGNORE_EXCEPTION_DETAIL' is in __all__ but not in stub
+doctest: 'NORMALIZE_WHITESPACE' is in __all__ but not in stub
+doctest: 'OutputChecker' is in __all__ but not in stub
+doctest: 'REPORTING_FLAGS' is in __all__ but not in stub
+doctest: 'REPORT_CDIFF' is in __all__ but not in stub
+doctest: 'REPORT_NDIFF' is in __all__ but not in stub
+doctest: 'REPORT_ONLY_FIRST_FAILURE' is in __all__ but not in stub
+doctest: 'REPORT_UDIFF' is in __all__ but not in stub
+doctest: 'SKIP' is in __all__ but not in stub
+doctest: 'UnexpectedException' is in __all__ but not in stub
+doctest: 'debug' is in __all__ but not in stub
+doctest: 'debug_src' is in __all__ but not in stub
+doctest: 'register_optionflag' is in __all__ but not in stub
+doctest: 'run_docstring_examples' is in __all__ but not in stub
+doctest: 'script_from_examples' is in __all__ but not in stub
+doctest: 'set_unittest_reportflags' is in __all__ but not in stub
+doctest: 'testfile' is in __all__ but not in stub
+doctest: 'testsource' is in __all__ but not in stub
+fcntl: 'DN_ACCESS' is in stub but is not defined at runtime
```

Comment:
These seem to be platform-specific flags that exist on various platforms. I don't think there's much hope of getting sys.platform checks to cover all this, so I'm planning to just ignore integer values in stubs that don't exist at runtime.